Composer/packages/client/src/App.tsx

```diff
@@ -123,13 +123,24 @@ const bottomLinks = [
 export const App: React.FC = () => {
   const { state, actions } = useContext(StoreContext);
   const [sideBarExpand, setSideBarExpand] = useState(false);
-  const { botName, projectId, dialogs, creationFlowStatus, locale, designPageLocation } = state;
+
+  const { botName, projectId, dialogs, creationFlowStatus, locale, designPageLocation, announcement } = state;
   const { setCreationFlowStatus } = actions;
   const mapNavItemTo = x => resolveToBasePath(BASEPATH, x);
 
   const openedDialogId = designPageLocation.dialogId || dialogs.find(({ isRoot }) => isRoot === true)?.id || 'Main';
   return (
     <Fragment>
+      <div
+        role="region"
+        aria-live="assertive"
+        style={{
+          position: 'absolute',
+          left: '-9999px',
```

Comment:
speaking from past experience, hiding the div way off to one side of the screen will cause scrollbar problems if/when RTL is brought into the mix. My suggestion would be to put it above the screen and/or change the div to 1x1 with no overflow to avoid the possibility.

Reply:
Thanks for the suggestion. I figured there'd be a better way to do that.